src/App.tsx

```diff
@@ -35,61 +36,117 @@ export function getReorderedGoods(
   const visibleGoods = [...goods];
 
   // Sort and reverse goods if needed
-  // eslint-disable-next-line no-console
-  console.log(sortType, isReversed);
+  if (sortType === SortType.ALPHABET) {
+    visibleGoods.sort((product1, product2) => (
+      product1.localeCompare(product2)
+    ));
+  }
+
+  if (sortType === SortType.LENGTH) {
+    visibleGoods.sort((product1, product2) => (
+      product1.length - product2.length
+    ));
+  }
```

Comment:
You can check condition inside callback which you provide to `sort` method.

Let's use `switch` case here. 
```suggestion
  visibleGoods.sort((product1, product2) => (
      switch (condition) {
        case :
        case :
        default:
      }
    ));
```